### PR TITLE
Update features list

### PR DESCRIPTION
**Pull request by jaapmarcus**
Probally should add some more info

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ title: Features
       <FeatureList :items="webDomains"></FeatureList>
     </template>
   </FeaturePageSection>
+  <FeaturePageSection image="/images/undraw_domain_names_re_0uun.svg">
+	  <template #title>DNS</template>
+	  <template #lead>Manage your own DNS server!</template>
+	  <template #list>
+		<FeatureList :items="dns"></FeatureList>
+	  </template>
+	</FeaturePageSection>
```
Comment (imjuniper) [inline docs/features.md]: there's some weirdness with the indentation. seems like it mixed spaces and tabs?

File: docs/_data/features.ts [hunk 2]
```diff
@@ -55,12 +56,21 @@ export const mail: FeatureListItem[] = [
...
+export const dns: FeatureListItem[] = [
+	{ text: "Create your own nameservers" },
+	{ text: "Easy setup for a DNS cluster" },
+	{ text: "Support for DNSSEC on domains" },
+];
+
 export const databases: FeatureListItem[] = [
-	{ text: "Support for MariaDB 10.2 -> 10.8 with 10.6 as default" },
+	{ text: "Support for MariaDB 10.2 -> 11.0 with 10.11 as default" },
+	{ text: "Support for Mysql 8" },
```
Comment (imjuniper) [inline docs/_data/features.ts]:
```suggestion
	{ text: "Support for MySQL 8" },
```

For consistency :)

File: docs/_data/features.ts [hunk 3]
```diff
@@ -55,12 +56,21 @@ export const mail: FeatureListItem[] = [
 	{ text: "Optional Rainloop installation" },
 ];
 
+export const dns: FeatureListItem[] = [
+	{ text: "Create your own nameservers" },
+	{ text: "Easy setup for a DNS cluster" },
```
Comment (imjuniper) [inline docs/_data/features.ts]:
```suggestion
	{ text: "Easy DNS cluster setup" },
```

Sounds a bit better to me

**Review by imjuniper — CHANGES_REQUESTED** [review]
There's some text I'd change a bit so it looks/sounds better. And there's also a bit of weirdness for some indentation.

I do agree, we probably should have a better/fuller feature page. We could ask ChatGPT for it 😆 But honestly though, I might ask a friend for help if she has time.

**Comment by imjuniper** [comment]
Also, can you replace the image with the same one, but with the Hestia pink colour (`#ff558f`), like the others have?

File: docs/_data/features.ts [hunk 4]
```diff
@@ -55,12 +56,21 @@ export const mail: FeatureListItem[] = [
...
+	{ text: "Support for Mysql 8" },
 	{ text: "Support for PostgreSQL" },
-	{ text: "Latest version of phpMyAdmin" },
+	{ text: "Latest version of phpMyAdmin and phpPgAdmin" },
 ];
 
 export const serverAdmin: FeatureListItem[] = [
-	{ text: "Automated backups to SFTP, FTP or Backblaze B2" },
+	{
+		text: "Automated backups to SFTP, FTP and via Rclone with 50+ (<a href='https://rclone.org/overview/'>Cloud storage providers</a>",
```
Comment (neto737) [inline docs/_data/features.ts]: It looks better without the parentheses

```suggestion
		text: "Automated backups to SFTP, FTP and via Rclone with 50+ <a href='https://rclone.org/overview/'>Cloud storage providers</a>",
```

**Comment by jaapmarcus** [comment]
Screenshot 2023-03-01 at 14 28 14

**Review by imjuniper — APPROVED** [review]

**Pull request merged**